backend/rocket-api/Cargo.toml

```diff
@@ -6,4 +6,5 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-rocket = "=0.5.0-rc.3"
```

Comment:
ew my brother was on my computer and did this

Reply:
:)